app/poros/film.rb

```diff
@@ -1,37 +1,39 @@
 class Film
-    attr_reader :title,
-                :original_title,
-                :api_id,
-                :overview,
-                :vote_average,
-                :adult,
-                :popularity,
-                :genres,
-                :runtime,
-                :reviews
+  attr_reader :title,
+              :original_title,
+              :api_id,
+              :overview,
+              :vote_average,
+              :adult,
+              :popularity,
+              :genres,
+              :runtime,
+              :reviews,
+              :poster
 
-    def initialize(data, cast = "nil", reviews = "nil")
-        @title = data[:title]
-        @original_title = data[:original_title]
-        @adult = data[:adult]
-        @api_id = data[:id]
-        @runtime = data[:runtime]
-        @popularity = data[:popularity]
-        @vote_average = data[:vote_average]
-        @overview = data[:overview]
-        @genres = data[:genres]
-        @cast = cast
-        @reviews = reviews
-    end
+  def initialize(data, cast = 'nil', reviews = 'nil')
+    @title = data[:title]
+    @original_title = data[:original_title]
+    @adult = data[:adult]
+    @api_id = data[:id]
+    @runtime = data[:runtime]
+    @popularity = data[:popularity]
+    @vote_average = data[:vote_average]
+    @overview = data[:overview]
+    @genres = data[:genres]
+    @cast = cast
+    @reviews = reviews
+    @poster = data[:poster_path]
```

Comment:
Did you add posters to the film_spec? - this is mostly a note for myself to add a line to that test.

Reply:
Working on posters!  It works on the films, but I think we need to add a column to the movies db to include the poster_path, then we can have posters attached to parties on the dashboard!